packages/vision/package.json

```diff
@@ -66,17 +66,22 @@
   "devDependencies": {
```

Comment:
I probably didn't, but they should not hurt anything; we should be merging Vision next week.